targets/TARGET_STM/USBPhyHw.h

```diff
@@ -27,6 +27,7 @@
 #include "platform/mbed_power_mgmt.h"
 #include "USBPhy.h"
 #include "PeripheralPins.h"
+#include "string.h"
```

Comment:
We need to revert this one, it's on master already (proper fix, we needed more headers)

Reply:
Fixed via https://github.com/ARMmbed/mbed-os/pull/13773

Reply:
fixed now here